integreat_cms/cms/models/media/media_file.py

```diff
@@ -202,6 +204,48 @@ def thumbnail_url(self):
             f"{settings.BASE_URL}{self.thumbnail.url}?{self.last_modified.timestamp()}"
         )
 
+    @property
+    def used_in(self):
+        """
+        Return where the media file is used.
+
+        :return: name and model of the contents in which the media file is used.
+        :rtype: str
+        """
+
+        as_icon = (
+            list(self.icon_organizations.all())
+            + list(self.icon_regions.all())
+            + list(self.events.all())
+            + list(self.pages.all())
+            + list(self.pois.all())
+        )
```

Comment:
This will produce a LOT of queries (5 queries per media element)...
Some regions have over 1000 media files in their library, which means this will cause 5k database queries and make the library veeeery slow. Please prefetch these reverse relationships for the whole queryset before using them on each individual item.